bin/pdagentd.py

```diff
@@ -404,6 +404,6 @@ def make_rotating_handler(fname, maxBytes, backupCount, level):
 
 
 # ---- Daemonize and run agent
-
-daemonize(pidfile, umask=_DEFAULT_UMASK)
+if not (len(sys.argv) > 1 and sys.argv[1] == '-f'):
+    daemonize(pidfile, umask=_DEFAULT_UMASK)
```

Comment:
I take it back - we're running it in systemd using `Type=simple` so we don't need to write a PIDFile. We should care about pidfile for non-systemd runs but that's out of scope here.